circle.yml

```diff
@@ -15,18 +15,12 @@ notify:
     - url: https://webhooks.stackstorm.net:8531/webhooks/build/events
 
 machine:
-  # Overwrite these ENV variables in parametrized (manual/API) builds
   environment:
     DISTROS: "wheezy jessie trusty xenial el6 el7"
     NOTESTS: "xenial el7"
     ST2_GITURL: https://github.com/StackStorm/st2
-    ST2_GITREV: master
```

Comment:
This makes it a bit less transparent now (for default values you need to check the scripts) so perhaps we should add a comment here or similar which points user to the right place with the default values,